sagemaker-huggingface/text-summarizer/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2626 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    curl https://arxiv.org/src/1911.12237v2/anc/corpus.7z --output corpus.7z
</pre><p>Preferred to host a copy of this dataset in the SageMaker S3 bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>